README.md

```diff
@@ -42,11 +42,11 @@ Maven:
 
     <dependency>
         <groupId>com.softwaremill.kmq</groupId>
-        <artifactId>core_2.12</artifactId>
+        <artifactId>core_2.13</artifactId>
         <version>0.2</version>
     </dependency>
 
-(Use `core_2.11` if you are using other components depending on Scala 2.11.)
+(Use `core_2.12` if you are using other components depending on Scala 2.12.)
```

Comment:
Maybe we should just suggest range of minor versions (to not change it every time, when we upgrade the lib)

E.g.
`(Use `core_2.XX` if you are using other components depending on Scala 2.XX, where XX is a minor version number)`

Reply:
I changed kind of the other way around: keeping the specific Scala versions but making the information explicitly about the supported Scala versions. What do you think?

Reply:
It's even better ;)